packages/@ama-sdk/generator-sdk/src/generators/shell/index.ts

```diff
@@ -36,13 +36,13 @@ export default class extends SdkGenerator {
     this.desc(`${deprecatedMessage}\nGenerate the shell of an SDK package`);
 
     this.option('projectName', {
-      description: 'Project name (NPM package scope)',
+      description: 'Project name (NPM package scope, package.json name will be @{projectName}/{packageName})',
```

Comment:
this package will be deprecated

Reply:
Indeed, but updating the description to have something more relevant doesn't cost much :D